cmd/tether/msgs/messages.go

```diff
@@ -76,6 +76,32 @@ var (
 	}
 )
 
+// PingMsg
+const PingReq = "ping"
+const PingMsg = "Ping"
+
+const UnblockReq = "unblock"
+const UnblockMsg = "Unblock"
+
+// VersionMsg
+const VersionReq = "version"
+
+type VersionMsg struct {
+	Version uint32
+}
+
+func (s *VersionMsg) RequestType() string {
+	return VersionReq
+}
+
+func (s *VersionMsg) Marshal() []byte {
```

Comment:
I believe we do but might be wrong. I was under the impression that ssh payloads are serialized byte arrays produced by ssh.Marshall